### PR TITLE
Implement basic layout of write-in transcription screen

### PR DESCRIPTION
**Pull request by taravancil**
## Overview
#1773 

This is a first pass at the layout for the write-in adjudication transcription screen. I am looking for feedback on the general approach, and whether there any UI tweaks we want to implement now (cc @beausmith).

It also updates two components:
- Updates padding on the `fullscreen` variant of the `Modal` component
- Adds an optional `screenTitle` prop to the `Navigation` component

## Demo Video or Screenshot
localhost_3000_write-ins (3)

## Testing Plan 
Added preliminary unit tests, manual QA. More tests to follow as I wire this up.

## Checklist
- [x] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: frontends/election-manager/src/screens/write_ins_transcription_screen.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+`;
+
+const Transcription = styled.div`
+  flex: 2;
+  position: relative;
+  border-left: 1px solid #cccccc;
+  padding: 1rem;
+`;
+
+const PreviouslyTranscriibedValuesContainer = styled.div``;
```
Comment (eventualbuddha) [inline frontends/election-manager/src/screens/write_ins_transcription_screen.tsx:25]:
```suggestion
const PreviouslyTranscribedValuesContainer = styled.div``;
```

**Review by eventualbuddha — COMMENTED** [review]
Mostly just noting some small code issues. I'll leave the style/UI approval to @beausmith.

File: libs/ui/src/main.tsx [hunk 2]
```diff
@@ -20,14 +20,15 @@ interface ChildProps {
   centerVertical?: boolean;
   centerHorizontal?: boolean;
   flexContainer?: boolean;
+  flexDirection?: string;
   maxWidth?: boolean;
```
[the remark this answers is not in the document]
Comment (beausmith) [reply libs/ui/src/main.tsx:24]: Let's save this for another PR. 😉

[the remark this answers is not in the document]
Comment (taravancil) [reply libs/ui/src/main.tsx:24]: I filed a tech-improvement ticket https://github.com/votingworks/vxsuite/issues/1813

**Comment by taravancil** [comment]
Stuck on getting the modal component's coverage to 100%, but this is ready for another round of review.

File: frontends/election-manager/src/screens/write_ins_transcription_screen.tsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    'Mickey Mouse',
+    'Mickey M',
+    'Micky',
+    'Donald',
+    'Donald Duck',
+    'Roger Rabbit',
+    'RR',
+    'DD',
+    'M. Mouse',
+  ];
```
[the remark this answers is not in the document]
Comment (taravancil) [reply frontends/election-manager/src/screens/write_ins_transcription_screen.tsx:61]: Probably, but we can save that for when this is actually wired up.

**Comment by taravancil** [comment]
@beausmith I suggest we merge this as-is, and then I will do #1826 once your PR with the overflow fix is merged.

**Comment by beausmith** [comment]
Sorry I caused these conflicts. Can I offer to rebase this on main and resolve the conflicts?

**Review by beausmith — APPROVED** [review]

**Pull request merged**